index.html

```diff
@@ -889,6 +884,79 @@ <h2 class='informative'>Vary</h2>
       </section>
     </section>
 
+    <section id='external-content'>
+      <h3>External LDP-NR Content</h3>
+      <p class='informative'>
```

Comment:
This `informative` annotation does not appear to have a visual effect in the rendered spec.